File: UnityProject/Assets/Scripts/MenuScripts/BackButton.js
#pragma strict
var image: GUITexture;
var Button: Texture2D;
var Button_Down: Texture2D;

function OnMouseEnter() {

	image.texture = Button_Down;
	
}

function OnMouseExit() {

	image.texture = Button;
	
}

function OnMouseDown() {

	Application.LoadLevel("MainMenu");
}

File: UnityProject/Assets/Scripts/MenuScripts/BackButton.cs
﻿using UnityEngine;
using System.Collections;

public class BackButton : MonoBehaviour {

	public AudioClip sound;
    public GUITexture image;
    public Texture2D Button;
    public Texture2D Button_Down;

    public void OnMouseEnter() {

		audio.PlayOneShot(sound);
	    image.texture = Button_Down;
	
    }

    public void OnMouseExit() {

	    image.texture = Button;
	
    }

    public void OnMouseDown() {

    	Application.LoadLevel("MainMenu");
    }

}
